uaclient/cli.py

```diff
@@ -103,26 +101,62 @@ def __init__(
         )
 
         self.base_desc = base_desc
-        self.non_beta_services_desc = non_beta_services_desc
-        self.beta_services_desc = beta_services_desc
 
     def error(self, message):
         self.print_usage(sys.stderr)
         self.exit(2, message + "\n")
 
     def print_help(self, file=None, show_all=False):
-        desc_vars = [
-            self.base_desc,
-            self.non_beta_services_desc,
-            self.beta_services_desc,
-        ]
-        if any(desc_vars):
-            services = sorted(self.non_beta_services_desc)
+        if self.base_desc:
+            non_beta_services_desc, beta_services_desc = (
+                self._get_service_descriptions()
+            )
+            services = sorted(non_beta_services_desc)
             if show_all:
-                services = sorted(services + self.beta_services_desc)
+                services = sorted(services + beta_services_desc)
             self.description = "\n".join([self.base_desc] + services)
         super().print_help(file=file)
 
+    @staticmethod
```

Comment:
Why this method need to be static ?

Reply:
Well it is an utility function inside the class, and it does not need the class itself or any instance to run. I kept it inside the class just because it is relevant only for a method this class particularly.